src/components/Pagination/styles.scss

```diff
@@ -0,0 +1,108 @@
+@import "../../styles/margins";
+@import "../../styles/colors";
+@import "../../styles/shadows";
+@import "../../styles/font-sizes";
+
+@mixin variants-common-styles($background-color, $border-color, $color) {
+    background-color: $background-color;
+    border: 1px solid $border-color;
+    color: $color;
+}
+
+.center-items {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.rainbow-pagination {
+    width: fit-content;
+}
+
+.rainbow-pagination_container {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    width: fit-content;
+}
+
+.rainbow-pagination_button {
+    height: 2rem;
+    width: 2rem;
+    border: 1px $color-gray-2 solid;
+    margin-left: -1px;
+    background-color: white;
+
+    @extend .center-items;
+
+    a {
```

Comment:
why you put styles to the ```a``` element, this is a bad practice instead add a className to this element